Add Firebase Admin module (#336)
quarkiverse/quarkus-google-cloud-services: docs/modules/ROOT/pages/firebase-admin.adoc

= Google Cloud Services - Firebase Admin

> This extension allows to inject both a `com.google.firebase.FirebaseApp` or even a `com.google.firebase.auth.FirebaseAuth` object inside your Quarkus application.

Reviewer: ```suggestion
This extension allows to inject both a `com.google.firebase.FirebaseApp` and a `com.google.firebase.auth.FirebaseAuth` object inside your Quarkus application.
```
